src/rules/seczetta-grab-risk-score.js

```diff
@@ -0,0 +1,223 @@
+  //  >>  'Content-Type':'application/json',
+  //  >>  'Authorization': 'Token token=${apiKey},
+  //  >>  'Accept': 'application/json'
+  //  Get Profile API 
+  //  >> get profile searches by 'name' this name field is configurable on the SecZetta side
+  //  >> Request:
+  //  >> https://taylordemo.mynonemployee.com/api/profiles?name=${name}
+  //  >>
+  //  >> Example Response: 
+  // >> "profiles": [
+  // >>     {
+  // >>         "id": "633b5e71-090c-4a47-a1a3-d0b8338df872",
+  // >>         "uid": "eedb47e4c2e147778a9e3be61c255a38",
+  // >>         "name": "testuser01@seczetta.com",
+  // >>         "profile_type_id": "5666f53e-cdd8-4420-8431-ca6e62e81451",
+  // >>         "status": "Active",
+  // >>         "updated_at": "2020-12-15T12:19:34.000-05:00",
+  // >>         "created_at": "2020-12-14T17:36:33.000-05:00",
+  // >>         "attributes": {
+  // >>             "personal_home_phone": "",
+  // >>             "engagement_contract_start_date": "12/14/2020",
+  // >>             "engagement_contract_end_date": "01/30/2021",
+  // >>             "personal_middle_name": "",
+  // >>             "personal_birthdate": "12/14/2020",
+  // >>             "engagement_type_of_employment": "Corporate Temp",
+  // >>             "personal_state": "AL",
+  // >>             "personal_mobile": "",
+  // >>             "personal_city": "",
+  // >>             "personal_email": "testuser01@seczetta.com",
+  // >>             "personal_zip": "",
+  // >>             "personal_first_name": "Test",
+  // >>             "personal_last_name": "User01",
+  // >>             "personal_street": ""
+  // >>         }
+  // >>     }
+  // >> ] 
+  //  Get Risk Score API 
+  //  >> notice in the response that the risk score still comes back as an array of 1
+  //  >> Request:
+  //  >> https://taylordemo.mynonemployee.com/api/risk_scores?object_id=${objectId}
+  //  >> 
+  //  >> Example Response: 
+  //  >> {
+  //  >>    "risk_scores": [
+  //  >>        {
+  //  >>            "id": "14118693-983e-462f-a330-f3b34d29f281",
+  //  >>            "uid": "036e7e2a3d0c41938609cdc6029d5b11",
+  //  >>            "object_id": "633b5e71-090c-4a47-a1a3-d0b8338df872",
+  //  >>            "object_type": "Profile",
+  //  >>            "overall_score": 3.5,
+  //  >>            "overall_risk_level_id": "29fc4382-2c60-4c6d-891b-15102cdc9e01",
+  //  >>            "impact_score": 7.0,
+  //  >>            "impact_risk_level_id": "29fc4382-2c60-4c6d-891b-15102cdc9e01",
+  //  >>            "probability_score": 0.0,
+  //  >>            "probability_risk_level_id": "c1f10e76-44c0-4bda-b344-8ae2a256d6c4"
+  //  >>        }
+  //  >>    ]
+  //  >> }
+
+  let profileResponse;
+  let riskScoreResponse;
+
+  let attributeId = configuration.SECZETTA_ATTRIBUTE_ID;
+  let profileTypeId = configuration.SECZETTA_PROFILE_TYPE_ID;
+
+  let uid = user.user_name || user.email; //depends on the configuration
```

Comment:
I think this should be `user.username`

https://auth0.com/docs/users/user-profile-structure